### ISSUE TITLE
Incorrect MIDI tempo with meter 2/2

### ISSUE THREAD

**Issue by p-himik** [paulrosen/abcjs#589]
Example ABC:
```
X:1
T:
V:1
M:2/2
L:1/8
Q:1/4=100
K:C
A B C
```
Plays twice as slow in both 6.0.0-beta.16 and 6.0.0-beta.24. And if you replace `M:2/2` with e.g. `M:4/4` it works perfectly fine.
Note that it works correctly at https://www.abcjs.net/abcjs-editor.html which apparently uses ABCJS 5.9.1

**Comment by paulrosen** [comment]
That's odd. It is working ok for me. 

And the key signature shouldn't matter because you are saying a 100 quarter notes are played in a minute.

Do you have a minimal example?

**Comment by p-himik** [comment]
It's as simple as it could possibly be:
`src/index.js`:
```
import 'abcjs/abcjs-midi.css';
import abcjs from "abcjs/midi";

const renderMidi = (meter) => {
    const node = document.createElement('div');
    document.getElementById('app').appendChild(node);
    const abc = `X:1
T:
V:1
M:${meter}
L:1/8
Q:1/4=100
K:C
A B C`;
    abcjs.renderMidi(node, abc);
}

renderMidi('4/4');
renderMidi('2/2');
```

`index.html`:
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" />
    <title></title>
</head>

<body>
<div id="app"></div>

<script src="src/index.js"></script>
</body>
</html>
```

`package.json`:
```
{
  "main": "index.html",
  "scripts": {
    "start": "parcel index.html --open",
    "build": "parcel build index.html"
  },
  "dependencies": {
    "abcjs": "6.0.0-beta.24"
  },
  "devDependencies": {
    "@babel/core": "7.2.0",
    "parcel-bundler": "^1.6.1"
  }
}
```

Run `npm install` and then `npm run start`.
The two MIDI controls will play at different speeds.

**Comment by paulrosen** [comment]
Oh, I see. You are using the old method of creating the midi which does have some bugs in it and I don't have time to maintain. 

Look at the example basic-synth.html for the new method. I'm in the process of improving the sound quality and functionality of this right now, too.

**Comment by p-himik** [comment]
Thanks! I didn't know `renderMidi` was deprecated. I see it now in the documentation but calling the method itself could also issue a warning in the JS console.

I'm not sure if I can use the new functionality just yet - created #594

**Comment by paulrosen** [comment]
There are some timing issues and some performance issues, particularly on mobile. There are also some problems with initializing the audioContext.

**Comment by paulrosen** [comment]
Fixed in version 6.0.0-beta.25

**Comment by p-himik** [comment]
This is again (still?) an issue on 6.0.0-beta.33.
If I go to https://editor.drawthedots.com/ and specify an explicit tempo, download MIDI, then change the time signature to `2/2` and download MIDI again, those two MIDI files will have different tempo. Same with the `6/8` signature and probably others.
